src/app/locales/resources.ts

```diff
@@ -0,0 +1,15 @@
+import extensionFR from './extension/fr-FR.json';
```

Comment:
```
const resources = {
  'fr-FR': {
    extension,
  },
  en: {
    extension,
  }
};
```

Following the resources object structure, it would feel more natural to store translations file under a `fr-FR` folder rather than an `extension` folder.

ex: `fr-FR/extension.json`
And then you could reexport all namespaces from an `fr-FR/index.ts` file and import namespaces here this way:
```
import { extension, profiles } from './extension/fr-FR';
```

Also since project structure will evolve toward #761 you should probably move `locales` folder to `src/locales`.